### PR TITLE
Add navigation bar and center text on index and essays pages

### PR DESCRIPTION
**Pull request by lyndapham**

### CODE REVIEW

File: lyndapham/step/portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
+
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
-#greeting-container {
+#fact-container {
   margin-top: 20px;
```
Comment (lyndapham) [inline lyndapham/step/portfolio/src/main/webapp/style.css:29]: comment

File: lyndapham/step/portfolio/src/main/webapp/essays.html [hunk 2]
```diff
@@ -4,12 +4,17 @@
     <meta charset="UTF-8">
     <title>My Essays</title>
     <link rel="stylesheet" href="style.css">
-    <!-- <script src="script.js"></script> -->
+    <script src="script.js"></script>
   </head>
   <body>
+        <div id="nav-bar">
```
Comment (andrewnguyen) [inline lyndapham/step/portfolio/src/main/webapp/essays.html]: Please use file-wide consistent indentation

File: lyndapham/step/portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -7,28 +7,36 @@
...
   </head>
   <body>
-    <div id="intro">
-      <h1>My Portfolio</h1>
-      <p>Hi there! I'm Lynda, a sophomore at the  University of Washington studying Computer Science.</p>
-      <p>Click here to get a random fact about me:</p>
-      <button onclick="addFunFact()">press</button>
-      <div id="fact-container"></div>
+    <div id="nav-bar">
+        <a href="index.html">Home</a>
```
Comment (andrewnguyen) [inline lyndapham/step/portfolio/src/main/webapp/index.html]: same here about consistent indentation

File: lyndapham/step/portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -1,9 +1,30 @@
+#nav-bar {
+    background-color: #edb245;
```
Comment (andrewnguyen) [inline lyndapham/step/portfolio/src/main/webapp/style.css]: here too

File: lyndapham/step/portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -49,5 +59,6 @@ <h1>Recent Projects</h1>
             width="5%" height="5%">
         </a>
     </div>
+    </div>
```
Comment (andrewnguyen) [inline lyndapham/step/portfolio/src/main/webapp/index.html:60]: extra </div> ?

File: lyndapham/step/portfolio/src/main/webapp/essays.html [hunk 6]
```diff
@@ -4,28 +4,33 @@
...
   </head>
   <body>
-        <main>
-            <h1>Lynda's Thoughts</h1>
-            <div class="content">
-                <div class="pdf">
-                    <h3>Driving</h3>
-                    <object width="360" height="270" data="pdfs/Driving.pdf"></object>
-                </div>
+      <div id="nav-bar">
```
Comment (andrewnguyen) [inline lyndapham/step/portfolio/src/main/webapp/essays.html]: The indentation is still inconsistent. Notice how above the indentation is 2 spaces but here is 4 spaces

Comment (lyndapham) [reply lyndapham/step/portfolio/src/main/webapp/essays.html]: Fixed! Sorry about that.

**Review by andrewnguyen — APPROVED** [review]

**Pull request merged**